cg/apps/demultiplex/sample_sheet/api.py

```diff
@@ -90,20 +90,12 @@ def _use_sample_sheet_from_housekeeper(self, flow_cell: FlowCellDirectoryData) -
             raise SampleSheetError(
                 f"Sample sheet for flow cell {flow_cell.id} does not exist in Housekeeper"
             )
-        try:
-            self.validate_sample_sheet(
-                sample_sheet_path=sample_sheet_path, bcl_converter=self.bcl_converter
-            )
-            LOG.info("Sample sheet from Housekeeper is valid. Copying it to flow cell directory")
-            if not self.dry_run:
-                link_or_overwrite_file(src=sample_sheet_path, dst=flow_cell.sample_sheet_path)
-        except SampleSheetError:
-            LOG.info(
-                f"Sample sheet {sample_sheet_path} failed validation, deleting from Housekeeper"
-            )
-            if not self.dry_run:
-                delete_file_from_housekeeper(file_path=sample_sheet_path, hk_api=self.hk_api)
-            raise SampleSheetError()
+        self.validate_sample_sheet(
+            sample_sheet_path=sample_sheet_path, bcl_converter=self.bcl_converter
+        )
+        LOG.info("Sample sheet from Housekeeper is valid. Copying it to flow cell directory")
+        if not self.dry_run:
+            link_or_overwrite_file(src=sample_sheet_path, dst=flow_cell.sample_sheet_path)
```

Comment:
Why do we not want to catch the sample sheet error?

Reply:
it will be caught in line 173 :)